tests/publiccloud/storage_perf.pm

```diff
@@ -75,7 +75,8 @@ sub run {
     my $disk_type = get_var('PUBLIC_CLOUD_HDD2_TYPE');
     my $url = get_var('PUBLIC_CLOUD_PERF_DB_URI');
     my $use_nvme = is_azure() && get_var('PUBLIC_CLOUD_INSTANCE_TYPE') =~ 'Standard_L(8|16|32|64)s_v(2|3)';
-
+    my $db_timeout = 300;
```

Comment:
```suggestion
    my $db_timeout = 30;
```
Isn't a timeout of 5 minutes for a database data push a little bit too much?

Reply:
We can obviously better tune it, but I wanted to let more time in the worst case: consider that the `wait` openqa command has `90s` as defult, IMHO here 30s is too small.
what if `180`?

Reply:
180s applied, if ok for everyone.